### PR TITLE
lint files

### PR DESCRIPTION
**Pull request by RomanTsegelskyi**
I recently noticed that `lintr` was not working for `pander` (there is a small bug in `lintr` that can be avoided by slightly changing code). And when I was reviewing some PRs recently, I noticed that we haven't been very consistent with our style. This is an attempt to fix that. I will lint other files when I have time

### CODE REVIEW

File: R/R5.R [hunk 1]
```diff
@@ -48,7 +48,7 @@
 #' ## Exporting to PDF (default)
 #' myReport$export()
 #'
-#' ## Or to docx in tempdir():
+#' # Or to docx in tempdir():
```
Comment (daroczig) [inline R/R5.R]: @RomanTsegelskyi I think we consistently & always used double hashes for standalone comments on their own and used single hash for inline comments (after actual R code). I'd be happy to keep that standard :)

**Comment by daroczig** [comment]
Please see above comment & failed Travis build -- but other than that, thank you very much for taking care of this, much appreciated!

**Comment by daroczig** [comment]
@RomanTsegelskyi any ideas on the Travis failure? I cannot reproduce that locally.

**Comment by RomanTsegelskyi** [comment]
Hmm, I was not able to reproduce it either. It might be change in knitr or other package, but I was not able to figure it out. I will probably get the VM to try to reproduce it in the exact environment.

**Comment by RomanTsegelskyi** [comment]
I was not even able to reproduce it on VM with Ubuntu 12.04 LTS. I am kind of out of ideas

**Comment by daroczig** [comment]
FTR I restarted the build yesterday on Travis to see if it was a temporary issue, but it failed with the same error message. Will try to find a way to debug this, but I don't have any ideas at the moment, either.

**Comment by daroczig** [comment]
@RomanTsegelskyi I disabled building the vignettes on the first build, which verified that this error comes from when doing hyphenation & we have some other failed tests as well at https://travis-ci.org/Rapporter/pander/builds/111564767

I will try to debug that hyphenation bug, but let me know if you have any further ideas.

**Comment by daroczig** [comment]
@RomanTsegelskyi good news on the strange error at https://github.com/travis-ci/travis-build/commit/f67c1010c9277a4607f31e514682fb29a82a9a7d#commitcomment-16611382 (it's fixed now)

Although the lint and GH page update parts still having some issues, but at least we are green again.

**Comment by RomanTsegelskyi** [comment]
Oh wow, thanks solving that. I will try to finish linting next week, after I come back to Seattle

**Comment by daroczig** [comment]
Thanks, @RomanTsegelskyi. I think the linting issue is now resolved after https://github.com/Rapporter/pander/commit/72c2f8983ea56e7d22b68ee036dc3df9cc7c3810

**Comment by RomanTsegelskyi** [comment]
@daroczig, I finished linting and rebased it on master. Good to merge?

File: R/S3.R [hunk 2]
```diff
@@ -815,80 +815,76 @@ pander.mtable <- function(x, caption = attr(x, 'caption'),
   coefs <- x$coefficients
   summaries <- x$summaries
 
-  num.models <- length(coefs)
```
Comment (daroczig) [inline R/S3.R]: Wow, this variable was never actually used? :)

File: R/pandoc.R [hunk 3]
```diff
@@ -1136,7 +1136,7 @@ pandoc.table.return <- function(t, caption, digits = panderOptions('digits'), de
 
     if (length(t.rownames) != 0) {
 
-        if ((length(split.cells) <= dim(t)[2]) && (length(split.cells) > 1)) {
+        if ( (length(split.cells) <= dim(t)[2]) && (length(split.cells) > 1)) {
```
Comment (daroczig) [inline R/pandoc.R]: Why the extra space here?

**Comment by daroczig** [comment]
Wow, thank you so much, @RomanTsegelskyi  -- this looks awesome! I left a very important :) question on a strange space between two parens (L1139), but I think it's OK to merge -- please feel free to do so.

**Pull request merged**